src/reconcile/rec_row.c

```diff
@@ -73,6 +92,80 @@ __rec_cell_build_int_key(WT_SESSION_IMPL *session, WTI_RECONCILE *r, const void
     return (0);
 }
 
+/*
+ * __wt_rec_pack_internal_key_addr --
+ *     Pack a key/value pair (internal page address or delta) directly into new_image.
+ */
+int
+__wt_rec_pack_internal_key_addr(WT_SESSION_IMPL *session, WT_ITEM *new_image,
```

Comment:
I would like these to be moved out of the rec module as well. We are not doing reconciliation. We should not have the rec wording in the naming.

Reply:
Then I think this may also need to change, we should have a separate function rather than __wti_rec_kv_copy in the reading path. I want the logic to completely decouple from reconciliation module.